### PR TITLE
Glyph cache disable

### PR DESCRIPTION
**Pull request by akallabeth**
deactivate glyph cache for good.

### CODE REVIEW

File: libfreerdp/core/settings.c [hunk 1]
```diff
@@ -272,53 +315,56 @@ rdpSettings* freerdp_settings_new(DWORD flags)
...
 	settings->EncryptionLevel = ENCRYPTION_LEVEL_NONE;
-
 	settings->CompressionEnabled = TRUE;
 
 	if (settings->ServerMode)
-			settings->CompressionLevel = PACKET_COMPR_TYPE_RDP61;
+		settings->CompressionLevel = PACKET_COMPR_TYPE_RDP61;
 	else
-			settings->CompressionLevel = PACKET_COMPR_TYPE_RDP61;
+		settings->CompressionLevel = PACKET_COMPR_TYPE_RDP61;
```
Comment (hardening) [inline libfreerdp/core/settings.c]: not your code, but CompressionLevel is always set to PACKET_COMPR_TYPE_RDP61

Comment (akallabeth) [inline libfreerdp/core/settings.c]: That is just the default, when using command line arguments you can change that.

**Comment by akallabeth** [comment]
@hardening already added the EXPERIMENTAL warning to the command line option, wouldn't disable it so that someone interested can spot and fix it.

File: libfreerdp/core/settings.c (hunk 1 above)
Comment (hardening) [inline libfreerdp/core/settings.c]: My point was more that the _if (settings->ServerMode)_ is useless as we're setting to PACKET_COMPR_TYPE_RDP61 in all cases.

Comment (bmiklautz) [inline libfreerdp/core/settings.c]: @hardening it still can be overridden so it still makes sense.

Comment (hardening) [inline libfreerdp/core/settings.c]: @bmiklautz well, my remark still applies: the _if (settings->ServerMode)_ is useless.

**Pull request merged**